demo/material/button.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+
+  <title>Material Theme Button</title>
+
+  <!-- Safari has severe rendering issues with native Shadow DOM
+       https://bugs.webkit.org/show_bug.cgi?id=170762 -->
+  <!-- <script>window.ShadyDOM = { force: true };</script> -->
+
+  <script src="../../../webcomponentsjs/webcomponents-loader.js"></script>
+  <link rel="import" href="common.html">
+
+  <link rel="import" href="../../material/material.html">
+  <link rel="import" href="../../material/elements/button.html">
+  <link rel="import" href="../../../vaadin-button/vaadin-button.html">
+
+
+  <custom-style>
+    <style>
+      [unresolved] {
+        opacity: 0;
+      }
+    </style>
+  </custom-style>
+</head>
+
+<body unresolved>
+
+  <div class="vertical-section-container centered">
+
+    <demo-navigation>
+      <h3 slot="header">Material theme</h3>
+    </demo-navigation>
+
+    <h1>Button</h1>
+
+    <h3>Normal</h3>
+    <nice-demo-snippet>
+      <script type="text/html" slot="source">
+        <vaadin-button>Normal</vaadin-button>
+        <vaadin-button disabled>Disabled</vaadin-button>
+        <vaadin-button theme="primary">Primary</vaadin-button>
```

Comment:
`<vaadin-button theme="primary">`

We don't have the `primary` theme, this syntax is confusing. Why not `<vaadin-button primary>`?